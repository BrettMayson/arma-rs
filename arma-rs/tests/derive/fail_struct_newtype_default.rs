use arma_rs::{FromArma, IntoArma};

#[derive(FromArma, IntoArma)]
#[arma(default)]
struct Container(u32);

#[derive(FromArma, IntoArma)]
struct Field(#[arma(default)] u32);

fn main() {}
