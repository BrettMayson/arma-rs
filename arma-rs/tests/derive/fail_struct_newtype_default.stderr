error: #[arma(default)] cannot be used on new type structs
 --> tests/derive/fail_struct_newtype_default.rs:4:8
  |
4 | #[arma(default)]
  |        ^^^^^^^

error: #[arma(default)] cannot be used on new type structs
 --> tests/derive/fail_struct_newtype_default.rs:8:21
  |
8 | struct Field(#[arma(default)] u32);
  |                     ^^^^^^^
